R/utils_report.R

```diff
@@ -7,11 +7,20 @@
 #' @param report_dir name of the folder where the report should be saved
 #'
 #' @importFrom quarto quarto_render
+#' @importFrom rstudioapi selectDirectory
 #' @export
 create_report <- function(report_params, report_name, report_dir) {
-  if (report_dir == "") {
-    message("The name specified for the report's directory is an empty string.")
-    report_dir <- "performance_report"
+  if (report_dir == "" || is.na(report_dir)) {
+    message(
+      "The name specified for the report's directory cannot be an empty string or NA. ",
+      "Make sure you're using RStudio"
+    )
+    report_dir <- selectDirectory(caption = "Please pick the report's directory")
```

Comment:
Maybe it will be easier if this function has only one parameter `output_file` instead of report_name and report_dir

Reply:
Good idea. `output_file` can be a blend of both report directory and report name. We need to make sure that the path (and file) given by the user actually exists.